sdk/appconfiguration/azure-appconfiguration-provider/MANIFEST.in

```diff
@@ -2,6 +2,8 @@ recursive-include tests *.py
 include *.md
 include LICENSE
 include azure/__init__.py
+include azure/appconfiguration/__init__.py
+include azure/appconfiguration/provider/*.py
 recursive-include samples *.py *.md
 recursive-include doc *.rst
-include azure/provider/py.typed
\ No newline at end of file
+include azure/appconfiguration/py.typed
```

Comment:
Shouldn't it be `azure/appconfiguration/provider/py.typed`?

Reply:
When it was under azure/appconfiguration/provider folder, we saw

INFO:root:Verifying presence of py.typed: [azure-appconfiguration-provider]
INFO:root:The py.typed file must exist in the base namespace for your package. Traditionally this would mean the furthest depth, EG 'azure/storage/blob/py.typed'.
INFO:root:Verified py.typed [azure-appconfiguration-provider]. Check messages above.